index.html

```diff
@@ -0,0 +1,160 @@
+<!-- <!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    Hello World!
+    <script  src="js/index.js"></script>
+  </body>
+</html> -->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+<body>
+	<main id="main">
+
+		<!-- Title of the page -->
+		<h1 id="title">
+			A. P. J. Abdul Kalam
+		</h1>
+
+		<div id="img-div">
+
+			<!--Image of the Tribute Person-->
+			<img src="https://upload.wikimedia.org/wikipedia/commons/6/6e/A._P._J._Abdul_Kalam.jpg" id="image"
+				alt="Error Loading Image">
+			<small id="img-caption">
+				Great Indian scientist and
+				politician who played a leading
+				role in the development of India’s
+				missile and nuclear weapons
+				programs.
+			</small>
+		</div>
+
+		<div id="tribute-info">
```

Comment:
For cleaner code, Please kindly wrap the "tribute-info" div into "section" tag and make "h1" tag into "header" tag. 👍

Reply:
- [x] Nice review here also: heavy_check_mark: